.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Pull request
+about: Create a pull request to contribute new features or bug fixes
+title: "[PULL REQUEST]"
+labels: pull_request
+
+---
+
+## Describe your changes
+
+## Related issue number (if any)
+
+## Checklist before requesting a review
+- [ ] I have performed a self-review of my code.
+- [ ] If it is a core feature, I have added thorough tests.
```

Comment:
This one feels vague. What's a core feature? What are thorough tests?

Reply:
Shouldn't we have a step for verifying that the UI works since we don't UI tests?

Reply:
revised language